src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,49 @@
 package core.basesyntax.service;
 
+import core.basesyntax.customexception.CustomException;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int VALID_LOGIN = 6;
+    private static final int VALID_PASSWORD = 6;
+    private static final int VALID_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkLoginValidation(user);
+        checkPasswordValidation(user);
+        checkAgeValidation(user);
+        return storageDao.add(user);
+    }
+
+    private static void checkLoginValidation(User user) {
+        if (user.getLogin() == null) {
+            throw new CustomException("Error! Login can't be null");
```

Comment:
```suggestion
            throw new CustomException("Login can't be null");
```
It isn't an error) Let’s remove it everywhere